### PR TITLE
create command for new banner page relations

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@saeedmb8 ready for review

File: gem/management/commands/create_new_banner_link_page_relations.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for main in Main.objects.all():
+            for banner in BannerPage.objects.descendant_of(main):
+                old_article_slug = ArticlePage.objects.get(
+                    pk=banner.banner_link_page).slug
+                new_article = ArticlePage.objects.descendant_of(
+                    main).get(slug=old_article_slug)
+                banner.banner_link_page = new_article
```
Comment (saeedmb8) [inline gem/management/commands/create_new_banner_link_page_relations.py]: @codiebeulaine we need to check here to see if the new_article exist and if not it should remove the banner_link_page and show a warning message

File: gem/management/commands/create_new_banner_link_page_relations.py [hunk 2]
```diff
@@ -6,12 +6,20 @@ class Command(BaseCommand):
...
-                if banner.live:
-                    banner.save_revision().publish()
-                else:
-                    banner.save_revision()
+                if banner.banner_link_page:
+                    old_article_slug = ArticlePage.objects.get(
+                        pk=banner.banner_link_page.pk).slug
+                    new_article = ArticlePage.objects.descendant_of(
+                        main).filter(slug=old_article_slug).first()
+                    print new_article.slug
```
Comment (saeedmb8) [inline gem/management/commands/create_new_banner_link_page_relations.py]: @codiebeulaine print statement

**Comment by saeedmb8** [comment]
🍰

**Pull request merged**